server/routes/form.js

```diff
@@ -667,6 +694,34 @@ module.exports = function Index({
     })
   )
 
+  router.post(
+    '/cancel/:bookingId',
+    asyncMiddleware(async (req, res, transactionalDbClient) => {
+      const { bookingId } = req.params
+      const formPageConfig = formConfig.cancel
+      const userInput = clearConditionalFields(req.body)
+
+      const valid = formService.isValid(formPageConfig, req, res, `/form/cancel/${bookingId}`, userInput)
+      if (!valid) {
+        return
+      }
+      if (userInput.confirm === 'Yes') {
+        await offendersService.setInactive(res.locals, bookingId, 'PENDING')
+
+        await formService.cancel({
```

Comment:
Is it missing some cancellation logging?

Reply:
added :)